package.json

```diff
@@ -40,7 +40,8 @@
     "*.toml": "pnpm run format:toml"
   },
   "engines": {
-    "pnpm": ">=7.25.0"
+    "pnpm": "8.2.0",
+    "node" : ">=16.14"
```

Comment:
Could add some context about why we need this requirement for `node`?

I'm not sure about this, you might want to split this change into a separate PR.

Reply:
> Could you add some context about why we need this requirement for `node`?
> 
> I'm not sure about this, you might want to split this change into a separate PR.

About this part its for 2 reasons:

1.  It is recommend using node 16 in the contribution doc so I though it would make sense to align the version in the CI as well
2. pnpm version 8 requires node version 16 ++